baremetal_benchmarking/workloads/tensorflow_workload.py

```diff
@@ -45,29 +45,43 @@ def download_bert_models(self):
             print("\n-- Downloading BERT checkpoints models..")
             utils.exec_shell_cmd(TF_BERT_CHECKPOINTS_WGET_CMD, None)
             utils.exec_shell_cmd(TF_BERT_CHECKPOINTS_UNZIP_CMD, None)
-            utils.exec_shell_cmd(TF_BERT_FP32_MODEL_WGET_CMD, None)
 
-        print("\n-- Required BERT models downloaded / already present..")
+        # We are deleting the model to avoid complication (with encryption in picture) on whether
```

Comment:
We may not need to delete the base model, as its the same version.

Reply:
Done. Earlier, was thinking to use the same path for encryption as well. But, then after changing the paths for plain and encrypted models, forgot to remove this snippet.